apps/todo/angular/src/app/todo/todo-item/todo-item.component.html

```diff
@@ -1,11 +1,24 @@
-<todo-checkbox
+<!-- <todo-checkbox
   [formControl]="checkbox"
   [checked]="todo.completed"
   (stateChange)="onToggleItem(checkbox.value)"
 >
   {{ todo.name }}
-</todo-checkbox>
+</todo-checkbox> -->
 
-<button mat-icon-button (click)="onRemoveItem()">
-  <mat-icon>clear</mat-icon>
-</button>
+<div class="form-control w-full max-w-xs justify-normal!">
```

Comment:
what is the reason for the exclamation point?

Reply:
no reason, is like !important in css but i just remove i was only testing that but still working with out

Reply:
> no reason, is like !important in css but i just remove i was only testing that but still working with out

like !important? it works?